src/openms/include/OpenMS/KERNEL/RangeManager.h

```diff
@@ -288,6 +288,7 @@ struct OPENMS_DLLAPI RangeRT : public RangeBase
 
   // Rule of 0!
   using RangeBase::RangeBase; // inherit C'tors from base
+  //using RangeBase::operator=; // inherit assignment operator from base
```

Comment:
@jpfeuffer @cbielow if I add this I get some ambigious operator= error

Reply:
Maybe something was added for RT meanwhile?
I am not sure why I needed it. But I'm pretty sure it did not compile without because the python wrappers use the assignment operator.

Reply:
@cbielow  is that even ok to expose the operstor= . I saw some conversion operators and a comment that they serve assignment.